### DISCUSSION TITLE
What will be new in Nitro 3?

### DISCUSSION THREAD

**Discussion by martinszeltins** [nitrojs/nitro#2586] [General]
I was looking at the roadmap for Nitro 3 and could not really understand what will be new or different in Nitro 3. What improvements will there be?

**Comment by pi0** [comment]
Initial release of Nitro v3 (itself) mainly includes breaking changes please see [#2521](https://github.com/unjs/nitro/pull/2521) for diff.

This allows us to unblock new possibilities from dependencies and for the future. jiti v2 with native ESM support, H3 v2 with native web support and much faster perf are some examples.

**Reply by martinszeltins** [reply]
Thank you 🙏🏻

**Comment by itshizhan** [comment]
When will it be released ? Is there a plan?
